lib/disco_log/presence.ex

```diff
@@ -90,6 +90,26 @@ defmodule DiscoLog.Presence do
     }
 
     {:ok, client} = WebsocketClient.send_event(client, identify_event)
+    {:noreply, %{state | websocket_client: client}, {:continue, :update_presence}}
+  end
+
+  # Update Presence after Hello event
+  # https://discord.com/developers/docs/events/gateway-events#update-presence
+  def handle_continue(
+        :update_presence,
+        %__MODULE__{websocket_client: client} = state
+      ) do
+    update_presence_event = %{
+      op: 3,
+      d: %{
+        activities: [%{type: 4, state: "🪩 Disco Logging", name: "Name"}],
```

Comment:
Theoretically, this could be just part of `Identify` event above, but for some reason this doesn't work for me. Identify can set online status, but not activities. I created a post in Discord developers server, maybe this is a bug or I missed something.